Userland/Libraries/LibWeb/CSS/StyleValues/CSSRGB.cpp

```diff
@@ -25,14 +25,14 @@ Color CSSRGB::to_color(Optional<Layout::NodeWithStyle const&>) const
             return normalized(style_value.as_number().number());
 
         if (style_value.is_percentage())
-            return normalized(style_value.as_percentage().value() * 2.55);
+            return normalized(style_value.as_percentage().value() * 255 / 100);
```

Comment:
Should this be `(value + 50) * 255 / 100` so that it rounds?

Reply:
`normalized()` is doing the rounding